packages/nx/src/command-line/migrate.ts

```diff
@@ -325,6 +341,32 @@ export class Migrator {
       .reduce((m, c) => ({ ...m, ...c }), {});
   }
 
+  private async runPackageJsonUpdatesConfirmationPrompt({
+    confirmationPrompt,
+    packages,
+  }: PackageJsonUpdates[number]): Promise<boolean> {
+    if (!confirmationPrompt) {
+      return Promise.resolve(true);
+    }
+
+    return await prompt([
+      {
+        name: 'shouldMigrate',
+        type: 'confirm',
```

Comment:
Yeah, I was kinda undecided about it. I went with No because when someone goes interactive, is to choose the opposite of the defaults. Still I think it makes sense to make them choose No consciously and not default to it. I'll change it.